src/components/panel/_macro.njk

```diff
@@ -1,43 +1,48 @@
 {% macro onsPanel(params) %}
-  {% if params is defined and params.classes is defined %}
-    {% set classes  = ' ' + params.classes %}
-  {% endif %}
+    {% if params.classes is defined %}
+        {% set classes  = ' ' + params.classes %}
+    {% endif %}
 
-  {% if params is not defined or params.title is not defined%}
-      {% set simpleClass = ' panel--simple' %}
-  {% endif %}
+    {% if params.title is not defined%}
+        {% set simpleClass = ' panel--simple' %}
+    {% endif %}
 
-  {% if params is defined and params.type is defined %}
-      {% set typeClass = ' panel--' + params.type %}
-  {% else %}
-      {% set typeClass = ' panel--info' %}
-  {% endif %}
+    {% if params.type is defined %}
+        {% set typeClass = ' panel--' + params.type %}
+    {% else %}
+        {% set typeClass = ' panel--info' %}
+    {% endif %}
 
-  {% if params is defined and params.spacious is defined %}
-      {% set spaciousClass = ' panel--spacious' %}
-  {% endif %}
+    {% if params.spacious is defined %}
+        {% set spaciousClass = ' panel--spacious' %}
+    {% endif %}
 
-  {% if params and params.icon %}
-      {% set iconClass = ' panel--icon' %}
-  {% endif %}
+    {% if params.iconsPath is defined and params.iconsPath %}
+        {% set iconsPath = params.iconsPath %}
+    {% else %}
+        {% set iconsPath = "/img/icons/" %}
+    {% endif %}
 
     <div
-        class="panel{{ typeClass }}{{ iconClass }}{{ simpleClass }}{{ spaciousClass }}{{ classes }}"
-        {% if params is defined and params.attributes is defined %}{% for attribute, value in (params.attributes.items() if params.attributes is mapping and params.attributes.items else params.attributes) %}{{attribute}}="{{value}}" {% endfor %}{% endif %}{% if params is defined and params.id is defined %} id="{{params.id}}"{% endif %}
+        class="{% if params.type != bare %}panel{% endif %}{{ typeClass }}{{ iconClass }}{{ simpleClass }}{{ spaciousClass }}{{ classes }}"
```

Comment:
Shouldn't the `panel` class be added in all scenarios as the parent class?

Reply:
Yeah true, the reason why I did that was because when using the `bare` type on the UAC pattern I kept getting a big gap because it was a gap added if you have one panel under another but ill try override that